include/mbedtls/aes.h

```diff
@@ -67,6 +67,22 @@
 /* MBEDTLS_ERR_AES_HW_ACCEL_FAILED is deprecated and should not be used. */
 #define MBEDTLS_ERR_AES_HW_ACCEL_FAILED                   -0x0025  /**< AES hardware accelerator failed. */
 
+#if defined( MBEDTLS_CHECK_PARAMS )
```

Comment:
Please move this to:
- a generic part in `platform_util.h`
- a module-specific part in `aes.c`, based on the generic part